lib/literal/enum.rb

```diff
@@ -199,7 +199,14 @@ def name
 	end
 
 	alias_method :inspect, :name
-	alias_method :to_s, :name
+
+	def to_s
+		name.demodulize.underscore.humanize
+	end
+
+	def to_sym
+		name.demodulize.to_sym
```

Comment:
I believe this one can be `self.class.names[self]` which is already a Symbol if I remember correctly.